src/app/components/application/program-info/program-info.component.css

```diff
@@ -36,4 +36,11 @@
 button.disabled {
   cursor: not-allowed;
 }
-/*# sourceMappingURL=program-info.component.css.map */
\ No newline at end of file
+/*# sourceMappingURL=program-info.component.css.map */
+/* apply only on iPad screens */
+@media (min-width: 350px) and (max-width: 1370px) {
+      .update-pending {
+          display: block;
+          margin-top: 5px; /* optional - add margin to separate link from text */
+      }
+}
```

Comment:
Confirm with @smithmd but I don't believe you should be modifying this .css file directly

Reply:
Bootstrap's classes can do all of these things without needing to use the `@media` styles.  
`display`: https://getbootstrap.com/docs/5.2/utilities/display/
`margin-top`: https://getbootstrap.com/docs/5.2/utilities/spacing/#margin-and-padding

We shouldn't need any new css to be written since these are already covered by bootstrap's classes:
`mt-md-1` sets the margin-top of medium sized screens and above to `0.25rem`

also, please start using `rem` instead of `px` as it handles the browser's zoom better than `px` does, and also allows people to change their own font size without breaking our layout.